pages/extensions/log_beautifier/log_beautifier.py

```diff
@@ -29,9 +29,22 @@ def get_logger(name='pod'):
 def print_server_ready_message(pod, host, port):
     home_doc = pod.get_home_doc()
     root_path = home_doc.url.path if home_doc and home_doc.exists else '/'
-    url = 'http://{}:{}{}'.format(host, port, root_path)
+    url_base = 'http://{}:{}/'.format(host, port)
+    url_root = 'http://{}:{}{}'.format(host, port, root_path)
     logger.LOGGER.info('Grow started successfully.')
-    return url
+
+    messages = manager.ServerMessages()
+    messages.add_message('Pod:', pod.root, colors.HIGHLIGHT)
+    messages.add_message('Server:', url_root, colors.HIGHLIGHT)
```

Comment:
Since the idea of this extension is to silence the startup log and we are not printing them anyway, this can be removed.